Scripts/scripts/cgal_create_CMakeLists

```diff
@@ -161,120 +153,35 @@ fi
     IFS=':'
     for cgal_component in $CGAL_COMPONENTS; do
       COMPONENT=`echo $cgal_component | tr '[:upper:]' '[:lower:]'`
-
-      # for qtmoc
-      if [ "$COMPONENT" = "qt4" ]; then
-        qt4='y'
-      fi
-
     done
     IFS=$OLDIFS
 
   fi
-
-  echo "find_package( CGAL QUIET COMPONENTS ${CGAL_COMPONENTS//:/ } )"
-
+  if [ -n "${CGAL_COMPONENTS}" ]; then
+    echo "find_package( CGAL REQUIRED OPTIONAL_COMPONENTS ${CGAL_COMPONENTS//:/ } )"
+  else
+    echo "find_package( CGAL REQUIRED )"
+  fi
   #vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
-  cat << 'EOF'
-
-if ( NOT CGAL_FOUND )
-
-  message(STATUS "This project requires the CGAL library, and will not be compiled.")
-  return()  
 
-endif()
-
-EOF
-  #---------------------------------------------------------------------------
-
-  #vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
-  cat << 'EOF'
+  if [ ! -z "$BOOST_COMPONENTS" ]; then
+    cat << 'EOF'
 
 # Boost and its components
 EOF
-  #---------------------------------------------------------------------------
-
-  if [ ! -z "$BOOST_COMPONENTS" ]; then
-
     echo "find_package( Boost REQUIRED COMPONENTS ${BOOST_COMPONENTS//:/ } )"
```

Comment:
Shouldn't it be Boost QUIET COMPONENTS  here ?

Reply:
Boost headers are required for CGAL. But components can be optional. Now that we require CMake 3.1, it could be:
```shell
echo "find_package( Boost REQUIRED OPTIONAL_COMPONENTS ${BOOST_COMPONENTS//:/ })"
```

Reply:
I pushed that modification.